src/main/java/com/google/sps/servlets/EditRecipeServlet.java

```diff
@@ -75,15 +76,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       recipeEntity.setProperty("ingredients", ingredients);
       recipeEntity.setProperty("steps", steps);
       recipeEntity.setProperty("tags", tags);
+      recipeEntity.setProperty("popularity", popularity);
       recipeEntity.setProperty("published", false);
 
       if (privacy.equals("public")) {
         recipeEntity.setProperty("published", true);
+        recipeEntity.setProperty("popularity", popularity++);
```

Comment:
What would happen if a user continually makes their recipe public, then private, and so on? Would they be able to make their recipe infinitely popular?

Reply:
wait no idk why I did that that doesn't make sense
I'm going to remove that